backend/server.js

```diff
@@ -16,12 +17,149 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+app.use(function(req, res, next) {
+  res.header('Access-Control-Allow-Origin', 'https://superb-alfajores-2745c6.netlify.app');
+  next();
+});
+
+
+const listEndpoints = require('express-list-endpoints');
+
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.json(listEndpoints(app));
+});
+
+// SCHEMA
+const { Schema } = mongoose;
+const UserSchema = new mongoose.Schema({
+  username: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  password: {
+    type: String,
+    required: true
```

Comment:
An idea would be to implement some limitations on password and username here, like minlength for instance? To practice handling that in both frontend and backend. As Daniel said, frontend and backend should never trust each other :)